build-support/ivy/ivysettings.xml

```diff
@@ -41,6 +42,11 @@ Licensed under the Apache License, Version 2.0 (see LICENSE).
            We setup the maven central resolver to require successful pom downloads here. -->
       <ibiblio name="maven-central" m2compatible="true" descriptor="required"/>
 
+      <!-- Custom repo for Kythe jars, as Google don't currently publish them anywhere. -->
```

Comment:
I feel like this is a blocker for landing this? But maybe not, since the effect would be that the contrib plugin needed a kythe install from "somewhere", and only the default would reference snowchain?